app/views/application/equality-monitoring/free-school-meals.html

```diff
@@ -1,31 +1,52 @@
-{% extends "_form.html" %}
+{% extends "layout.html" %}
 {% set parent = "Equality and diversity questions" %}
-{% set title = "If you finished school after 1980, were you eligible for free school meals at any point during your school years?" %}
+{% set title = "Did you ever get free school meals in the UK?" %}
 
 {% block pageNavigation %}
   {{ govukBackLink({
     href: paths.back
   }) }}
 {% endblock %}
 
-{% block primary %}
-  {{ govukRadios({
-    items: [{
-      text: "Yes"
-    }, {
-      text: "No"
-    }, {
-      text: "Not applicable, I finished school before 1980 or went to school overseas"
-    }, {
-      text: "I don't know"
-    }, {
-      divider: "or"
-    }, {
-      text: "Prefer not to say"
-    }]
-  } | decorateApplicationAttributes(["equality-monitoring", "free-school-meals"])) }}
-
-  {{ govukButton({
-    text: "Continue"
-  }) }}
+{% block content %}
+
+  <form{% if formaction %} action="{{ formaction }}"{% endif %} method="post" novalidate>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
```

Comment:
Do you normally use `govuk-grid-column-two-thirds-from-desktop`? I'd be using it for nearly all two column grids.